tools/installer/full-node.sh

```diff
@@ -223,10 +302,28 @@ EOF
     print_color $GREEN "Systemd service set up and started successfully."
 }
 
+# Function to check if ufw is installed and open port 26656. We need to open the port to keep the network healthy.
+# By default, at least on Debian vultr, this port is not open to the internet.
+configure_ufw() {
+    if command -v ufw &> /dev/null; then
+        print_color $YELLOW "ufw is installed."
+        read -p "Do you want to open port 26656 for p2p communication? (Y/n): " open_port
+        if [[ $open_port =~ ^[Yy] ]]; then
+            ufw allow 26656
+            print_color $GREEN "Port 26656 opened successfully."
+        else
+            print_color $YELLOW "Port 26656 not opened."
```

Comment:
Does landing in this conditional branch really guarantee that the port is not open? If not, I would suggest we rephrase the message to be more precise (e.g. `"Firewall rules not modified"`).